docs/apis/http/webdav/_index.md

```diff
@@ -0,0 +1,533 @@
+
+{{< hint type=caution title="Use the Depth header with caution" >}}
+**Depth: infinity**
+
+Using the `Depth: infinity` header value can cause heavy load on the server, depending on the size of the file tree.
+
+The request can run into a timeout and the server performance could be affected for other users.
+
+{{< /hint >}}
+
+## Create a Directory
+
+Clients create directories (WebDAV collections) by executing a `MKCOL` request at the location specified by the request url.
+
+{{< tabs "create-folder" >}}
+{{< tab "Curl" >}}
+```shell
+curl -L -X MKCOL 'https://localhost:9200/dav/spaces/storage-users-1%24some-admin-user-id-0000-000000000000/NewFolder/' \
+-H 'Authorization: Basic YWRtaW46YWRtaW4='
+```
+{{< /tab >}}
+{{< tab "HTTP" >}}
+```shell
+MKCOL /dav/spaces/storage-users-1%24some-admin-user-id-0000-000000000000/NewFolder/ HTTP/1.1
+Host: localhost:9200
+Authorization: Basic YWRtaW46YWRtaW4=
+```
+{{< /tab >}}
+{{< /tabs >}}
+### Response
+
+{{< tabs "response create folder" >}}
+{{< tab "201 - Created" >}}
+This indicates that the Resource has been created successfully.
+
+#### Body
+
+The response has no body.
+{{< /tab >}}
+{{< tab "403 - Forbidden" >}}
+
+#### Body
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<d:error xmlns:d="DAV" xmlns:s="http://sabredav.org/ns">
+    <s:exception>Sabre\DAV\Exception\Forbidden</s:exception>
+    <s:message></s:message>
+</d:error>
+```
+{{< /tab >}}
+{{< tab "405 - Method not allowed" >}}
+
+#### Body
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<d:error xmlns:d="DAV" xmlns:s="http://sabredav.org/ns">
+    <s:exception>Sabre\DAV\Exception\MethodNotAllowed</s:exception>
+    <s:message>The resource you tried to create already exists</s:message>
+</d:error>
+```
+{{< /tab >}}
+{{< /tabs >}}
+
+## Upload File
+
+To upload files to the remote server, clients can use the `PUT` method to create or fully replace the content of the remote file.
+
+### Request Headers
+
+| Name          | Usage                                                                                                                                                                                                                                                                                                               |
+|---------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `X-OC-Mtime`  | Send the last modified <br> time of the file to the server in unixtime format.                                                                                                                                                                                                                                      |
```

Comment:
```suggestion
| `X-OC-Mtime`  | Send the last modified <br> time of the file to the server in unixtime format.  The server applies this mtime to the resource rather than the actual time.                                                                                                                                                                                                                                   |
```

Reply:
ok